src/components/Modals/EditSemester.vue

```diff
@@ -135,13 +128,14 @@ export default Vue.extend({
 
   &-button {
     width: 4.75rem;
-    height: 2rem;
-    color: $sangBlue;
+    height: 1.8rem;
+    color: $activeGray;
     border-radius: 3px;
-    border: 1px solid $sangBlue;
+    border: 1px solid $primaryGray;
     background-color: $white;
     display: flex;
     justify-content: center;
+    align-items: center;
```

Comment:
I believe it should be `color: $sangBlue` and `border: 1px solid $sangBlue`. Is there a reason why the colors were changed?

This is an example of a modal design from Figma:
![Screen Shot 2021-03-18 at 11 02 11 AM](https://user-images.githubusercontent.com/54298311/111648061-665dac00-87d9-11eb-860c-2aedcf456dcf.png)